drivers/acid-transactions.rst

```diff
@@ -0,0 +1,12 @@
+ACID transactions guarantee atomicity, consistency, isolation, and durability so
+that database operations are executed safely, even when failures occur. ACID
+transactions are required in some industries where errors or partial updates
+could lead to serious financial, legal, or safety consequences. 
+
+By default, operations on single documents conform to ACID standards. For
+operations on multiple documents or collections, MongoDB supports multi-document
+ACID transactions using sessions. These transactions use the snapshot isolation
+level so applications see a consistent set of data while the transaction is
```

Comment:
This is a nit, but a small suggestion to possibly better explain what we mean by "snapshot isolation".

```suggestion
ACID transactions using sessions. Sessions operate on data from a single point-in-time snapshot, so applications see a consistent set of data while the transaction is
```